proposals/NNNN-swiftpm-extensible-build-tools.md

```diff
@@ -534,6 +540,8 @@ let package = Package(
 
 As with the previous example, the `using` parameter of the executable target applies the `SwiftProtobuf` extension for that target.
 
+[Tom] didnt we want to leave this for configuration files for the initial release? I worry about commiting to too much API surface area upfront
```

Comment:
Yes, agreed, though I'm 100% sure this will come up in the discussion.  It seemed reasonable to allow strings for now and then deprecate that in favor of nice types later, rather than not have a solution at all.  But it might be best to leave this out for now.  I'll take it out.